teachertool/src/components/HeaderBar.tsx

```diff
@@ -151,27 +151,42 @@ export const HeaderBar: React.FC<HeaderBarProps> = () => {
             <></>
         );
         return (
-            <div className={css["user-menu"]}>
-                {teacherTool.userProfile ? (
-                    <MenuDropdown
-                        id="profile-dropdown"
-                        items={items}
-                        label={avatarElem || initialsElem}
-                        title={lf("Profile Settings")}
-                    />
-                ) : (
+            <>
+                <div>
                     <Button
-                        className={classList("inverted", css["sign-in-button"])}
-                        rightIcon="xicon cloud-user"
-                        title={lf("Sign In")}
-                        label={lf("Sign In")}
+                        className={css["feedback-btn"]}
+                        labelClassName="min-sm"
+                        leftIcon="xicon feedback"
+                        title={Strings.GiveFeedback}
+                        label={Strings.GiveFeedback}
                         onClick={() => {
-                            pxt.tickEvent(Ticks.UserMenuSignIn);
-                            showModal({ modal: "sign-in" });
+                            pxt.tickEvent(Ticks.FeedbackForm);
                         }}
+                        href="https://aka.ms/teachertool-feedback"
```

Comment:
Does this go anywhere yet? I was trying to visit it just in browser and just kept hitting bing.

Reply:
Nope! We still have to create the survey form.